micro_sam/sam_annotator/image_series_annotator.py

```diff
@@ -64,7 +71,7 @@ def image_series_annotator(
     """Run the 2d annotation tool for a series of images.
```

Comment:
This should be change to reflect that the 3d annotation tool is now also supported.